spring-petclinic-messaging-emulator/src/main/resources/application.yml

```diff
@@ -1,5 +1,11 @@
-spring:
-  application:
-    name: messaging-emulator
-  # config:
-  #   import: optional:configserver:${CONFIG_SERVER_URL:http://localhost:8888/}
+spring:
+  application:
+    name: messaging-emulator
+  config:
+    import: optional:configserver:${CONFIG_SERVER_URL:http://localhost:8888/}
+  sql:
```

Comment:
this is taken from application.yml in the config repo

Reply:
how can it take it if not yet configured the config server? the other services have the same config, see this for instance https://github.com/Azure-Samples/spring-petclinic-microservices/blob/labstarter/spring-petclinic-vets-service/src/main/resources/application.yml